spec/lib/rbac/filterer_spec.rb

```diff
@@ -830,6 +870,18 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
             expect(objects).to eq(targets)
           end
 
+          it "returns the correct results when targets are of different types" do
```

Comment:
maybe this should be `"returns the correct class for different classes of targets"`